templates/console/source/lambda/config-version/cf_config_version_manager_graphql/cf_config_version_manager_graphql.py

```diff
@@ -546,43 +586,50 @@
 
     return result
 
+def get_config_version(ddb_table, distribution_id, src_version):
+    response = ddb_table.get_item(
+        Key={
+            "distributionId": distribution_id,
+            "versionId": int(src_version)
+        })
+    data = response['Item']
+    if not data:
+        raise Exception(f"Failed to get the version with distribution id:{distribution_id}, version_id:{src_version}")
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Using `Exception` and `BaseException` can make your code prone to errors and difficult to maintain. Instead, we recommend using one of the [Built-in Exceptions](https://docs.python.org/3/library/exceptions.html) or creating a custom exception class that is derived from `Exception` or one of its subclasses.